actors/builtin/paych/paych_actor.go

```diff
@@ -323,10 +314,8 @@ func (pca Actor) Collect(rt vmr.Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
 	)
 	builtin.RequireSuccess(rt, codeTo, "Failed to send funds to `To`")
 
-	rt.State().Transaction(&st, func() interface{} {
-		st.ToSend = big.Zero()
-		return nil
-	})
+	rt.DeleteActor(st.From)
```

Comment:
Great, just add a comment saying "return remaining balance to From upon deletion"